src/authorizations/components/redesigned/TokenRow.tsx

```diff
@@ -0,0 +1,134 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect, ConnectedProps} from 'react-redux'
+import {createDateTimeFormatter} from 'src/utils/datetime/formatters'
+
+// Actions
+import {
+  deleteAuthorization,
+  updateAuthorization,
+} from 'src/authorizations/actions/thunks'
+
+// Components
+import {
+  ComponentSize,
+  FlexBox,
+  AlignItems,
+  FlexDirection,
+  JustifyContent,
+  ComponentColor,
+  Button,
+  ResourceCard,
+  IconFont,
+} from '@influxdata/clockface'
+
+import {Context} from 'src/clockface'
+
+// Types
+import {Authorization} from 'src/types'
+import {
+  DEFAULT_TOKEN_DESCRIPTION,
+  UPDATED_AT_TIME_FORMAT,
+} from 'src/dashboards/constants'
+
+import {relativeTimestampFormatter} from 'src/shared/utils/relativeTimestampFormatter'
```

Comment:
do we not also want to use `createRelativeFormatter` from `'src/utils/datetime/formatters'` ?

Reply:
hmm..  `relativeTimestampFormatter` is actually using `createRelativeFormatter` in order to return the date in _from now_ format. here is what `relativeTimestampFormatter` looks like
![Screen Shot 2021-07-30 at 5 02 32 PM](https://user-images.githubusercontent.com/66275100/127715903-0ab673c9-8c52-406d-98d5-cb9663b865f8.png)

Reply:
haha, I can't even remember my own code that I wrote 👴